extra/onnx_ops.py

```diff
@@ -17,7 +17,7 @@ def Add(input: Tensor, other: Tensor, broadcast=None): return input + other if i
 def Sub(input: Union[Tensor, Any], other: Tensor): return input - other # some test has input as int
 def Mul(input: Tensor, other: Tensor): return (input * other) if input.dtype == dtypes.float else (input * other).cast(input.dtype)
 # in openpilot, due to SHUFFLE_PAD_OPS issues, we are spending an extra kernel
-def Div(input: Tensor, other: Tensor): return input / other if input.dtype == dtypes.float else input.div(other).floor()
+def Div(input: Tensor, other: Tensor): return input / other if input.dtype == dtypes.float or (isinstance(input.dtype, ImageDType) and input.dtype.name == "imagef") else input.div(other).floor()
```

Comment:
I don't understand why there's ever a floor here

Reply:
right - the onnx spec sorta expects that for non-floats
https://github.com/onnx/onnx/blob/main/docs/Operators.md#examples-43
but I dont see why there'd be a need for a floor if the dtype is already an int. I think you also saw that the image stuff breaks because of this

Reply:
same - The onnx spec sorta does `//` instead of `/` for a uint8 div https://github.com/onnx/onnx/blob/main/docs/Operators.md#examples-43 you also saw how it breaks the image dtypes